charts/woodpecker/charts/agent/values.yaml

```
@@ -40,7 +40,8 @@ secrets: {}
 #      WOODPECKER_AGENT_SECRET: ${WOODPECKER_AGENT_SECRET}
 
 # -- Additional volumes that can be mounted in containers
-extraVolumes: []
+extraVolumes:
+  []
```

This formatting is wrong. Sometimes cause by prettier due to the wrong indentation in the comment block below.

```suggestion
extraVolumes: []
```

---

Yes it is a prettier thing. I wonder if there is a setting which could prevent this?

---

Ah nice, removing/fixing the indentation did the trick 👍